packages/core/src/meta/MetaManagerImpl.ts

```diff
@@ -99,15 +102,15 @@ export default class MetaManagerImpl extends MetaManager {
   /**
    * @inheritdoc
    */
-  setLink(relation: string, value: string) {
-    this._link.set(relation, value);
+  setLink(relation: string, reference: MetaValue, otherAttrs?: MetaAttributes) {
```

Comment:
Can we rename `relation` to `rel`, `reference` to `href`? So it matches the actual attributes.

(don't forget on the MetaManager class)